cpp/oneapi/dal/algo/jaccard.hpp

```diff
@@ -19,4 +19,4 @@
 
 #pragma once
 
-#include "oneapi/dal/algo/jaccard/vertex_similarity.hpp"
+//#include "oneapi/dal/algo/jaccard/vertex_similarity.hpp"
```

Comment:
I guess it's not the best practice to comment such things. It would be better to remove this